src/router/index.ts

```diff
@@ -55,28 +55,30 @@ const routes: RouteRecordRaw[] = [
     redirect: '/login',
   },
   {
-    path: '/app',
-    redirect: appStore.appId ? '/app/dashboard' : '/app/create',
+    path: '/apps',
+    redirect: appStore.appId
```

Comment:
Is the plan to remove this later when add the "manage apps" page?

Reply:
Yeah we'll remove redirect and place the component for manage apps